synapse/tests/test_lib_gis.py

```diff
@@ -0,0 +1,32 @@
+from synapse.tests.common import *
+
+import synapse.lib.gis as s_gis
+
+# earth mean radius in mm
+r = 6371008800
+
+ratios = {
+    'cm': 10.0,
+    'm': 1000.0,
+    'km': 1000000.0,
+}
+
+km = 1000000.0 # using mm as base units
+gchq = (51.8994, -2.0783)
+
+class GisTest(SynTest):
+
+    def test_lib_gis_haversine(self):
```

Comment:
I think that we should add some additional tests for haversine distance.

Reply:
cool. what kind of tests did you have in mind?

Reply:
Was discussing this with @vEpiphyte this morning. They are adding the additional tests to this PR.